src/Coalesce.Web.Vue/Controllers/HomeController.cs

```diff
@@ -1,17 +1,20 @@
-using System;
-using System.Collections.Generic;
 using System.Diagnostics;
-using System.Linq;
-using System.Threading.Tasks;
+using System.IO;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.FileProviders;
 
-namespace VuePlayground.Controllers
+namespace Coalesce.Web.Vue.Controllers
 {
     public class HomeController : Controller
     {
         public IActionResult Index()
         {
-            return File("~/index.html", "text/html");
+
+            IFileProvider provider = new PhysicalFileProvider(Path.Combine(Directory.GetCurrentDirectory(),"wwwroot"));
```

Comment:
Can this be changed to use `IHostingEnvironment.WebRootPath` instead of assuming "wwwroot"? See https://blog.mariusschulz.com/2016/05/22/getting-the-web-root-path-and-the-content-root-path-in-asp-net-core

Reply:
Capital idea, sir!